lib/tasks/temporary/in_business_data_migration.rake

```diff
@@ -0,0 +1,15 @@
+desc 'Covert in_business to a string'
```

Comment:
Yeah, once I've run these data migrations, I'm going to have a PR that removes them. They'll always be in the git history, but I don't want them lingering around.